guides/internationalization.md

```diff
@@ -6,11 +6,12 @@ The first step is to read and follow the [Rails I18n Guide](http://guides.rubyon
 
 ## Localizing a slice
 
-Firstly add `localize: true` to all text fields which need localzing.
+Firstly add `localize: true` to all text fields which need localizing, make sure you add a 
+default value as there is a bug in Mongoid which means the value may not be set correctly.
```

Comment:
Probably good to put a link to the bug here too (as well as commit message). That way users can see for themselves if they need to do it (it might get fixed in future 4.x release).